src/runtime/hashmap.go

```diff
@@ -15,8 +15,8 @@ type hashmap struct {
 	buckets    unsafe.Pointer // pointer to array of buckets
 	seed       uintptr
 	count      uintptr
+	valueSize  uint32
 	keySize    uint8 // maybe this can store the key type as well? E.g. keysize == 5 means string?
-	valueSize  uint8
```

Comment:
was this order change intentional?

Reply:
Yes. It reduces padding for the struct. If we change the keySize type too then we can put it back.

Reply:
Ok, that's reasonable.